software/cloud-dashboard/react-client/src/components/temp-view/temp-plot.js

```diff
@@ -9,6 +9,8 @@ import Input from 'react-toolbox/lib/input';
 const LINE_COLORS = ['#e41a1c', '#377eb8', '#4daf4a', '#984ea3', '#ff7f00', '#ffff33']
 
 const defaultNumberOfPoints = 300;
+const tempMin = 0;
```

Comment:
since these changes will be in the generalized `plot` component as described above, these should likely be input `props` to the component (perhaps `minLimit` and `maxLimit`). These values can be passed into the `Plot` component by the parent, as is seen [here](https://github.com/mlp6/ADPL/blob/master/software/cloud-dashboard/react-client/src/components/plot-card.js#L35). Inside the `Plot` component, you can of course access the current prop value using syntax similar to the following `this.props.minLimit`